test/file.jl

```diff
@@ -244,8 +244,22 @@ close(s)
     end
 end
 
-my_tempdir = tempdir()
-@test isdir(my_tempdir) == true
+@testset "tempdir" begin
+    my_tempdir = tempdir()
+    @test isdir(my_tempdir)
+    @test my_tempdir[end] != '/'
+    @test my_tempdir[end] != '\\'
+
+    var =  Sys.iswindows() ? "TMP" : "TMPDIR"
+    PATH_PREFIX = Sys.iswindows() ? "C:\\" : "/tmp/"
+    MAX_PATH = (Sys.iswindows() ? 251 : 1024) - length(PATH_PREFIX) - 1 # null-termination character
```

Comment:
For reference I'm pretty sure there's a bug in the libuv function that hits an edge case that shouldn't happen, which is why the test here is for 251 chars and does not test longer paths.

hitting this branch I believe https://github.com/libuv/libuv/blob/c4e9657d59723a6273ce68bda935680e1b3596c5/src/win/util.c#L1186
Looks like the issue is that the code is using MAXPATH+1 instead of MAXPATH